src/CMakeLists.txt

```diff
@@ -10,3 +10,11 @@ install(PROGRAMS ks8851-set-mac revpi-factory-reset piSerial
 
 install(FILES revpi-factory-reset.sh
   DESTINATION /etc/profile.d)
+
+add_custom_target(lan9500.bin ALL
```

Comment:
Wouldn't it be better named lan95xx? As the commit message says it is for the lan9514.

Reply:
maybe it should be named exactly as lan9514.hex and lan9514.bin, as in the image, the product id fields are already specified as 0x9514.
But for the script, it might be proper to name it as 95xx to align with the name of driver "smsc95xx", we can add options to differentiate the variants of 95xx by options if it will be needed in the future.